virtwho/virt/vdsm/jsonrpc.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import unicode_literals
+
+"""
+Module for JSON-RPC over STOMP, part of virt-who
+
+Copyright (C) 2018 Kevin Howell <khowell@redhat.com>
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+
+import json
+import ssl
+import uuid
+
+from virtwho.virt.vdsm.stomp import StompClient
+
```

Comment:
I suspect you aware of `python-jsonrpclib` module and I understand reasons why it is not used. Please add note (link) here about the module with explanation why it is not used.